shopelectro/settings/prod.py

```diff
@@ -1,74 +0,0 @@
-from .base import *
-
-
-DEBUG = False
-
-ALLOWED_HOSTS = ['*']
-
-if not os.environ.get('TEST_ENV', False):
-    # http->https change
-    os.environ['HTTPS'] = 'on'
-
-    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
-
-    CSRF_COOKIE_SECURE = True
-    SESSION_COOKIE_SECURE = True
-
-YANDEX_KASSA_LINK = 'https://money.yandex.ru/eshop.xml'
-
-USE_CELERY = True
-
-REDIS_URL = os.environ['REDIS_URL']
-REDIS_PORT = os.environ['REDIS_PORT']
-REDIS_PASSWORD = os.environ['REDIS_PASSWORD']
-
-REDIS_DSN = f'redis://{REDIS_URL}:{REDIS_PORT}/'
-
-REDIS_LOCATION_DEFAULT = os.environ['REDIS_LOCATION_DEFAULT']
-REDIS_LOCATION_SESSION = os.environ['REDIS_LOCATION_SESSION']
-REDIS_LOCATION_THUMBNAIL = os.environ['REDIS_LOCATION_THUMBNAIL']
-REDIS_LOCATION_USER_AGENT = os.environ['REDIS_LOCATION_USER_AGENT']
-
-CACHES = {
```

Comment:
I did not understand the point. Where will we store these configs?

Reply:
somewhere out of cvs. Simple - on serv and every of us in personal vault. Complex - separated vault or separated cvs repository with configs.

We should remove it by two reasons:
- we should be able to change local (serv too) config without working with cvs
- we should be able to put some secrets to this config

Reply:
> Simple or Complex

We should decide which one to choose. Let's create todo for this

Reply:
created https://github.com/fidals/shopelectro/issues/332